### DISCUSSION TITLE
How to override ManagedPrivateEndpoint array values in the config file

### DISCUSSION THREAD

**Discussion by Magro28** [Azure-Player/azure.datafactory.tools#298] [Q&A]
Hello,
I have the following managedPrivateEndpoint in our ADF:
```
{
	"name": "adls_mpe_sap_adf",
	"properties": {
		"privateLinkResourceId": "/subscriptions/xxxxx/resourceGroups/rg-xxxxx-westeurope/providers/Microsoft.Storage/storageAccounts/xxxxxx",
		"groupId": "dfs",
		"fqdns": [
			"xxxxxx.dfs.core.windows.net"
		]
	}
}
```

I try to override the fqdns value, with this config:
```
managedPrivateEndpoint,kv_mpe_sap_adf,$.properties.fqdns,"kvdevsagadfsapweu.vault.azure.net"
```
but I get the following error:
The term 'kvdevsagadfsapweu.vault.azure.net' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

How is the right path and syntax to override the value? Especially because it is an array in the json.

Best,
Mario

**Comment by NowinskiK** [comment]
**fqdns** is an array, so you need to refer to the (first) element of the array.
This one should work:
```
managedPrivateEndpoint,adls_mpe_sap_adf,$.properties.fqdns[0],"kv3.vault.azure.net"
```
